app/src/main/java/bg/zahov/app/util/Extensions.kt

```diff
@@ -137,8 +139,8 @@ fun String.parseTimeStringToLong(): Long {
 
 fun Exercise.toExerciseSetAdapterWrapper(units: Units = Units.METRIC): ExerciseSetAdapterExerciseWrapper {
     return ExerciseSetAdapterExerciseWrapper(
-        noteVisibility = if(note.isNullOrEmpty()) View.GONE else View.VISIBLE,
-        note =  note,
+        noteVisibility = if (note.isNullOrEmpty()) View.GONE else View.VISIBLE,
```

Comment:
In the previous comment I meant these also look redundant and not needed. We don't need such specifics like `View.VISIBLE` in a UI State holder. We can use a boolean, or better yet check if the data is available in the object and show loading/error if it is not.